src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -120,31 +120,27 @@ export const submitFormFeedback: RequestHandler<
   }
 
   // Form is valid, proceed to next step.
-  const submitFeedbackResult = await PublicFormService.insertFormFeedback({
+  return PublicFormService.insertFormFeedback({
     formId: form._id,
     rating,
     comment,
   })
-
-  if (submitFeedbackResult.isErr()) {
-    const { error } = submitFeedbackResult
-    logger.error({
-      message: 'Error creating form feedback',
-      meta: {
-        action: 'handleSubmitFeedback',
-        ...createReqMeta(req),
-        formId,
-      },
-      error,
+    .map((submitFeedbackResult) =>
+      res.status(StatusCodes.OK).json(submitFeedbackResult),
```

Comment:
let's not call this variable `Result`, since one might mistake it for a `neverthrow` `Result`, which it's not

Reply:
removed as the response is no longer returned